### PR TITLE
Hgg subtype update: Adding tp53 status

### PR DESCRIPTION
**Pull request by kgaonkar6**
⚠️ merge after #942 is in master

#### What scientific question is your analysis addressing?
Adding TP53 status along with evidence columns(tp53_XYZ from classifier,SNV and CNV calls to HGG samples.

#### What was your approach?
I am using tp53_nf1_score/results/tp53_altered_status.tsv to add the tp53_altered status to the subtypes already annotated to the sample_id

#### What GitHub issue does your pull request address?
#807 

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/molecular-subtyping-HGG/10-HGG-TP53-annotation.Rmd [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: "TP53 annotation for HGG"
+output: html_notebook
+---
+
+In this notebook we will annotate HGG samples with TP53 status we obtained from snv/cnv and tp53 loss classifier. We belive TP53 annotation will add useful information to the current known subtypes as we see Tp53 mutated co-occuring with H3 mutations
```
Comment (jharenza) [inline analyses/molecular-subtyping-HGG/10-HGG-TP53-annotation.Rmd:6]:
```suggestion
In this notebook we will annotate HGG samples with TP53 status we obtained from snv/cnv and TP53 classifier. We believe TP53 annotation will add useful information to the current known subtypes as we see TP53 mutations co-occurring with H3 mutations.
```

**Review by jharenza — CHANGES_REQUESTED** [review]
I think that we want to capture any TP53-altered samples in the subtyping, rather than just loss. For `HGG_molecular_subtype.tsv`, will you also add `TP53 activated`?

**Comment by kgaonkar6** [comment]
> I think that we want to capture any TP53-altered samples in the subtyping, rather than just loss. For `HGG_molecular_subtype.tsv`, will you also add `TP53 activated`?
will do!

**Review by jharenza — APPROVED** [review]
looks great, thank you!

**Pull request merged**